teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>PDBQT and PDB should be capitalized when mentioned in normal text, or wrapped with backticks if lowercase.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>